src/explore-education-statistics-common/src/services/publicationService.ts

```diff
@@ -206,4 +210,11 @@ export default {
       `/publications/${publicationSlug}/releases/${releaseSlug}/prerelease-access-list`,
     );
   },
+  getPublications(
+    params: GetPublicationsRequest,
+  ): Promise<PublicationsResponse> {
+    return contentApi.get(`/publications?`, {
```

Comment:
This shouldn't need a `?` at the end. Think Axios should automatically append the `params` into a query string without any further intervention.

Reply:
yeah that's typo, it's gone now.